src/Rings.jl

```diff
@@ -167,6 +167,25 @@ function is_square_with_sqrt(a::RingElem)
   end
 end
 
+###############################################################################
+#
+#   Ring properties
+#
+###############################################################################
+
+is_perfect(F::Field) = characteristic(F) == 0 || F isa FinField ||
+                                                 throw(NotImplementedError())
+
+function is_finite(F::Field)
+  if F isa FinField
+    return true
+  elseif characteristic(F) == 0
+    return false
+  else
+    throw(NotImplementedError())
+  end
+end
```

Comment:
Maybe use method dispatch?
```suggestion
is_finite(F::FinField) = true

function is_finite(F::Field)
  if characteristic(F) == 0
    return false
  else
    throw(NotImplementedError())
  end
end
```
or even this
```suggestion
is_finite(F::FinField) = true

is_finite(F::Field) = characteristic(F) != 0 && throw(NotImplementedError())
```
I think I find the former version clearer. I can live with any of the three versions.